libpod/image/image.go

```diff
@@ -496,6 +500,32 @@ func (i *Image) History() ([]ociv1.History, []types.BlobInfo, error) {
 	return oci.History, img.LayerInfos(), nil
 }
 
+// Dangling returns a bool if the image is "dangling"
+func (i *Image) Dangling() bool {
+	dangling := false
```

Comment:
Can simplify to `return len(i.Names()) == 0`

Reply:
fixed